css/style.css

```diff
@@ -60,4 +60,92 @@
 .button--danger {
 	color: #ffffff;
 	background-color: #ce4141;
-}
\ No newline at end of file
+}
+/*Comodities */
+html {
+  font-family: "Roboto", sans-serif;
+  margin: 20px;
+}
+@media (max-width: 768px) {
```

Comment:
Even with this media query being well implemented it is not needed, please remove it.
`@media (min-width: 768px) {` should be declared for the desktop css once it is implemented.